app/src/org/commcare/dalvik/activities/CommCareSetupActivity.java

```diff
@@ -215,13 +220,14 @@ protected void onCreate(Bundle savedInstanceState) {
             this.ccApp = oldActivity.ccApp;
         }
 
-        Log.v("UiState","Current vars: " +
+        Log.v("UiState", "Current vars: " +
                         "UIState is: " + this.uiState + " " +
                         "incomingRef is: " + incomingRef + " " +
                         "startAllowed is: " + startAllowed + " "
         );
 
         uiStateScreenTransition();
+        performSMSInstall(false);

```

Comment:
I think so - the goal of this is to basically make install as seamless as possible and since we have no ability to configure per domain at this point I think we basically have to have this here if we want automatic installs